targets/unity-v2/source/ExampleTestProject/Assets/PlayFabSDK/Shared/Public/PlayFabAuthenticationContext.cs

```diff
@@ -6,7 +6,7 @@ public PlayFabAuthenticationContext()
         {
         }
 
-        public PlayFabAuthenticationContext(string clientSessionTicket, string entityToken, string playFabId, string entityId, string entityType) : this()
+        public PlayFabAuthenticationContext(string clientSessionTicket, string entityToken, string playFabId, string entityId, string entityType, string telemetryKey = null) : this()
```

Comment:
I understand adding he telemetry key to PlayFabAuthenticationContext, but do you think it might also make sense to include it in PlayFabSettings? Similar to DeveloperSecretKey: 
https://github.com/PlayFab/UnitySDK/blob/master/ExampleTestProject/Assets/PlayFabSDK/Shared/Public/PlayFabSettings.cs#L91